identity-provider/src/pages/Dashboard.jsx

```diff
@@ -0,0 +1,171 @@
+    }
+    return (
+        <>
+        <h1>You're not currently connected to any organizations.</h1>
+        <p>Search for voulenteer oppertunities <button type = "submit" onClick = {voulenteerButton}> Here</button></p>
+        
+        </>
+    )
+}
+
+function DashboardOrganizations() {
+
+   // code copied from material ui: Will show the dialog component on the website
+  //  Will need a seperate one for each dialog's title and description
+  //  Will need another open, setOpen for the other dialog
+
+
+   const [open, setOpen] = React.useState(false);
+   const [dialogTitle] = React.useState('Hello World');
+    //  open is the current state value( in dialog, open = {open}), setOpen is the function that updates the state and we are setting it to true 
+  //  useState returns an array
+ 
+  // open is set to false. What does false do though? Maybe we can change the true and false statements to something else?
+  // if you were to call setOpen(true), it will change the value of open to true
+  // so how does this carry over to what we have going on so far?
+
+
+   const handleClickOpen = () => {
+     setOpen(true);
+   };
+ 
+   const handleClose = () => {
+     setOpen(false);
+   };
+ // *********************************************
+
+//  We want dialog to show up for each button, so do we need to put the return inside the buttons or keep it in the function?
+    const editInfoButton = () => {
+        console.log('This button will bring up a typed out alert which shows your information and contains x butttons to delete info with a cancel and save button.')
+    }
+
+    const unlinkButton = () => {
+        console.log('This button will bring up a typed alert which will ask you if you want to unlink from the organization the user pressed, with a cancel and confirm button.')
+    }
+    return (
+        <>
+        <h1>Your Volunteer Organizations</h1>
+        <h2>Select an account to sign in </h2>
+        <a href = 'https://www.redcross.org/'></a>
+        <button type = 'submit' onClick = {editInfoButton}> View Info</button>
+        {/* We need a patch request for the unlink button to update the list of organizations once the user presses 'confirm' */}
+        <button type = 'submit' onClick = {unlinkButton}>Unlink</button>
+
+        {/* This Dialog will come up after you click the "Edit Info" button. */}
+
+        <Dialog
+        open={open}
+        onClose={handleClose}
+        aria-labelledby={dialogTitle}
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogTitle id="alert-dialog-title">
+          {"Your Information"}
+        </DialogTitle>
+        <DialogContent>
+          <DialogContentText id="alert-dialog-description">
+            {/* Placeholder unordered list for the list of organizations: Will want to show what user's information is linked to those organizations*/}
+            <p> You are currently sharing this information with ______</p>
+          <ul>
```

Comment:
This would pull the permissions from the organizations state object using the currentOrganization to pull the specific organization info.